README.md

```diff
@@ -50,5 +54,18 @@ make dockerbuild-go
 ./cli_wallet
 ```
 
+By default, the wallet will attempt to connect to a locally running Spacemesh full node using the default node's grpc api port.
+You can configure the ports in go-sm and use CLIWallet `--grpc-server` and `--grpc-port` flags to override the defaults.
+To use the full wallet features, enable all GRPC services in your node's config file. e.g:
```

Comment:
Probably best not to document this here, this documentation should live under go-spacemesh and should link to it here.

Reply:
It is also there but this is useful. Without it, it is hard to run the CLIWallet for devs who work on it. It wasn't obvious for me..